src/manager/RealPrintManager.test.ts

```diff
@@ -1,60 +1,25 @@
 import * as printerLib from 'printer'
 import uuid from 'uuid/v1'
+import fakeFile from '../../test/utils/fakeFile'
 import mockOf from '../../test/utils/mockOf'
 import RealPrintManager from './RealPrintManager'
-import { PrintJobState, File } from '.'
+import { PrintJobState } from '.'
```

Comment:
I'd like to know more about this strategy of placing types in the `index.ts` file. Is this common? Should we move to this?

Reply:
It's not so much putting types in `index.ts` as just putting them alongside the classes/functions/etc that "own" them, if that makes sense. `src/manager/index.ts` exports `getPrintManager` and essentially "owns" the concept of a print manager, so it also exports the `PrintManager` interface and related types.

Think of it like this: you probably wouldn't have a `types.java` or `types.cpp` or `types.swift` in "real" typed languages, so why do that in JavaScript/TypeScript?